lib/consyncful/item_mapper.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 module Consyncful
+  ##
+  # Responsible for mapping an update recieved from Contentfuls
+  # syncronisation API into useful fields for consyncful.
```

Comment:
I wonder if it'd be good to mention that it's for mapping them into useful fields to be used by `Consyncful::PersistedItem` to create/update/delete the entry in mongo? Or if that's all implied.